amundsen_application/static/js/containers/SearchPage/index.tsx

```diff
@@ -9,7 +9,7 @@ import SearchPage, { DispatchFromProps, StateFromProps } from '../../components/
 
 export const mapStateToProps = (state: GlobalState) => {
   return {
-    searchTerm: state.search.searchTerm,
+    searchTerm: state.search.search_term,
```

Comment:
fun fact: there are `camelCase` and `decamelize` libraries that we can consider using in the future to get rid of this discrepancy. More boiler plate but I think at least that logic can be quarantined to `api.ts` folders when we parse the response from python. I'll file a ticket so we don't lose the idea, and can revisit later when feature demands die down.